app/src/server/fineTuningProviders/openpipe/trainingConfig.ts

```diff
@@ -45,9 +46,17 @@ export async function trainingConfig(fineTune: TypedFineTune): Promise<AxolotlCo
     where: { id: fineTune.projectId },
   });
 
-  const trainingEntries = await prisma.fineTuneTrainingEntry.count({
-    where: { fineTuneId: fineTune.id },
-  });
+  const trainingEntries = await kysely
+    .selectFrom("FineTuneTrainingEntry as fte")
```

Comment:
Nit: I've been aliasing FineTuneTrainingEntries as `ftte`, and I think we should standardize on that.

Reply:
Will change.